### PR TITLE
Add rocketfuel data source

### PR DESCRIPTION
**Pull request by StefanGreffenius**
## before merge

- [x] set `uri:` to ENV var

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -40,6 +40,12 @@
 
   c.default_cassette_options = { allow_unused_http_interactions: false }
 
+  c.before_record do |i|
+    i.request.headers.delete('X-Auth-Token')
+    i.response.headers.delete('Set-Cookie')
+    i.response.headers.delete('X-Auth-Token')
+  end
```
Comment (tsujigiri) [inline spec/spec_helper.rb]: :+1:

File: spec/integration/import_rocket_fuel_data_into_postgres_spec.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  let(:rocketfuel_data_source) do
+    Importeur::DataSources::Rocketfuel.new(rocketfuel_service)
+  end
+
+  let(:rocketfuel_service) do
+    RocketfuelApi::Service::Company.new(rocketfuel_connection)
+  end
+
+  let(:rocketfuel_connection) do
+    # TODO: Change to production API endpoint
```
Comment (guigs) [inline spec/integration/import_rocket_fuel_data_into_postgres_spec.rb]: What's wrong with using sandbox api for testing?

File: lib/importeur/data_sources/rocketfuel.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Importeur
+  module DataSources
+    class Rocketfuel
+      def initialize(rocketfuel_service)
+        @rocketfuel_service = rocketfuel_service
```
Comment (guigs) [inline lib/importeur/data_sources/rocketfuel.rb]: See my suggestion in https://github.com/ad2games/importeur/pull/1/files#r138452203. So we could get rid of that `and_wrap_original` in the test.

File: spec/integration/import_rocket_fuel_data_into_postgres_spec.rb (hunk 2 above)
Comment (StefanGreffenius) [reply spec/integration/import_rocket_fuel_data_into_postgres_spec.rb]: Good question, just thought would be good to take the real endpoint. But in the end it doesnt matter i guess.

**Comment by guigs** [comment]
Some Rocketfuel services, like `line_item` [1] have pagination. `get_all` should iterate for all pages to get all items. Also, it has a required filter (campaign_id), so there should be a way to pass params to `get_all`.

But if we are only interested in companies service now, it should be ok. Just wanted to point it out so we keep that in mind when we need to use with other services.

[1] https://api.rocketfuel.com/doc/2016/index.html?raml=%2F2016%2Fdocs%2Fline_item

**Comment by StefanGreffenius** [comment]
Thats true we need to keep that in mind and adjust it when we need. In the end, only the companies `GET` requests are used so far (and spec'ed). We didn't focused at the other stuff and added this to the rocketfuel_api readme: `Note: Only the Company Service GET requests are tested so far.`

@guigs @tsujigiri Okay to merge for now?

**Pull request merged**